devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusNative.java

```diff
@@ -420,7 +420,7 @@ public void buildNative() {
             @Override
             public void accept(ConfigBuilder configBuilder) {
                 InMemoryConfigSource type = new InMemoryConfigSource(Integer.MAX_VALUE, "Native Image Type")
-                        .add("quarkus.package.types", "NATIVE");
+                        .add("quarkus.package.types", "native");
                 configBuilder.withSources(type);
```

Comment:
You also need to add the `configs` below to the `configBuilder`

Reply:
Yup, done

Reply:
After the 10th try :P

Reply:
Don't forget to remove this line then :)

Reply:
:+1: